editor/selectors.js

```diff
@@ -122,44 +245,124 @@ export function getSelectedBlocks( state ) {
 	return blockOrder.slice( startIndex, endIndex + 1 );
 }
 
+/**
+ * Returns the unique ID of the block which begins the multi-selection set, or
+ * null if there is no multi-selectino.
+ *
+ * @param  {Object}  state Global application state
+ * @return {?String}       Unique ID of block beginning multi-selection
+ */
 export function getBlockSelectionStart( state ) {
-	return state.multiSelectedBlocks.start;
+	return state.multiSelectedBlocks.start || null;
 }
 
+/**
+ * Returns the unique ID of the block which ends the multi-selection set, or
+ * null if there is no multi-selectino.
+ *
+ * @param  {Object}  state Global application state
+ * @return {?String}       Unique ID of block ending multi-selection
+ */
 export function getBlockSelectionEnd( state ) {
-	return state.multiSelectedBlocks.end;
+	return state.multiSelectedBlocks.end || null;
 }
 
+/**
+ * Returns an array containing all unique IDs of the post being edited, in the
+ * order they appear in the post.
+ *
+ * @param  {Object} state Global application state
+ * @return {Array}        Ordered unique IDs of post blocks
+ */
 export function getBlockUids( state ) {
 	return state.editor.blockOrder;
 }
 
+/**
+ * Returns the index at which the block corresponding to the specified unique
+ * ID occurs within the post block order, or `-1` if the block does not exist.
+ *
+ * @param  {Object} state Global application state
+ * @param  {String} uid   Block unique ID
+ * @return {Number}       Index at which block exists in order
+ */
 export function getBlockOrder( state, uid ) {
```

Comment:
Perhaps separately, I think this selector should be renamed. As-is, I'd expect it to simply​ return the block order array. `getBlockIndex` maybe?